action/src/run.ts

```diff
@@ -70,15 +64,16 @@ export const run = async () => {
     setFailed(
       'Visual tests failed to execute successfully. Perhaps one failed to take a screenshot?'
     );
-    return octokit.rest.repos.createCommitStatus({
-      sha: commitHash,
-      context: VISUAL_REGRESSION_CONTEXT,
-      state: 'failure',
-      description: VISUAL_TESTS_FAILED_TO_EXECUTE,
-      ...context.repo
-    });
+    return createCommitStatus(
+        commitHash,
+        'failure',
+        VISUAL_TESTS_FAILED_TO_EXECUTE
+    );
```

Comment:
I wrapped it in common method so that we can print the error message => `Failed to update commit status.`
This would give little more clarity on reason why job failed.